docker-centos/Dockerfile

```diff
@@ -2,11 +2,10 @@ FROM centos
 
 LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>"
 
-RUN yum install -y docker docker-latest container-selinux python-docker-py docker-lvm-plugin docker-rhel-push-plugin docker-novolume-plugin lvm2 iptables procps-ng xz cloud-utils-growpart && yum clean all
+RUN yum install -y docker docker-latest docker-selinux python-docker-py docker-lvm-plugin docker-rhel-push-plugin docker-novolume-plugin lvm2 iptables procps-ng xz cloud-utils-growpart && yum clean all
```

Comment:
yes, I've added a new patch after the "Revert patch" that updates a few things including the `s|docker-selinux|container-selinux|` package rename